peps/pep-0794.rst

```diff
@@ -7,6 +7,7 @@ Type: Standards Track
 Topic: Packaging
 Created: 05-Jun-2025
 Post-History: `02-May-2025 <https://discuss.python.org/t/90506>`__
+              `05-Jun-2025 <https://discuss.python.org/t/94567>`__
```

Comment:
Please also add to `Discussions-To`

Reply:
Oops! 😅 https://github.com/python/peps/pull/4452